README.md

```diff
@@ -348,7 +395,69 @@ use a different path for this file:
 _answers_file: .my-custom-answers.yml
 ```
 
-### Updating a project
+### Template helpers
+
+In addition to
+[all the features Jinja supports](https://jinja.palletsprojects.com/en/2.10.x/templates/),
+Copier includes:
+
+#### Builtin variables/functions
+
+- `now()` to get current UTC time.
+- `make_secret()` to get a random string.
+- `_copier_answers` includes the current answers dict, but slightly modified to make it
+  suitable to [autoupdate your project safely](#the-answers-file):
+  - It doesn't contain secret answers.
+  - It doesn't contain any data that is not easy to render to JSON or YAML.
+- `_copier_conf` includes the current copier `ConfigData` object, also slightly
+  modified:
+  - It only contains JSON-serializable data.
+  - But you have to serialize it with `[[ _copier_conf.json() ]]` instead of
+    `[[ _copier_conf|tojson ]]`.
+  - ⚠️ It contains secret answers inside its `.data` key.
+  - Modifying it doesn't alter the current rendering configuration.
+
+#### Builtin filters
+
+- `anything|to_nice_yaml` to print as pretty-formatted YAML.
+
+  Without arguments it defaults to:
+  `anything|to_nice_yaml(indent=2, width=80, allow_unicode=True)`, but you can modify
+  those.
+
+**Warning:** Use only trusted project templates as these tasks run with the same level
+of access as your user.
```

Comment:
This comment is important, but it doesn't make sense in this chapter.

Reply:
I moved it just below, at the beginning of `Generating a project`.